backend/open_webui/routers/users.py

```
@@ -58,31 +85,88 @@ async def get_user_permissisions(user=Depends(get_verified_user)):
     return Users.get_user_groups(user.id)
 
 
+############################
+# Get Users Count
+############################
+
+
+@router.get("/count")
+async def get_users_count(domain: str = None, user=Depends(get_verified_user)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)